VideoLocker/res/values/strings.xml

```diff
@@ -260,6 +260,8 @@
     <string name="post_attribution">{time} by {author} {author_label}</string>
     <!-- The date and author of an answer to a discussion question e.g. "Marked as answer 2 days ago by Khalid TA" -->
     <string name="answer_author_attribution">Marked as answer {time} by {author} {author_label}</string>
+    <!-- The date and author of an endorsed response to a discussion thread e.g. "Endorsed 2 days ago by Khalid TA" -->
+    <string name="endorsing_author_attribution">Endorsed {time} by {author} {author_label}</string>
```

Comment:
Since the endorser is separate from the author, `endorsing_author` should probably be changed to just `endorser`.